Lib/gftools/builder/__init__.py

```diff
@@ -306,6 +323,36 @@ def build_static(self):
             self.mkdir(self.config["woffDir"], clean=True)
         if self.config["buildTTF"]:
             self.build_a_static_format("ttf", self.config["ttDir"], self.post_process_ttf)
+            if "instances" in self.config:
+                self.instantiate_static_fonts(
+                    self.config["ttDir"], self.post_process_ttf
+                )
+            else:
+                self.build_a_static_format(
+                    "ttf", self.config["ttDir"], self.post_process_ttf
+                )
```

Comment:
The first `self.build_a_static_format()` can be removed. I think it comes from a rebase/merge conflict.

Reply:
Good catch. Thanks.